src/main.rs

```diff
@@ -130,7 +130,7 @@ fn run(
     let mut display = Display::new(&config, options, logger_proxy.clone())?;
 
     info!(
-        "PTY Dimensions: {:?} x {:?}",
+        "PTY Dimensions: `{:?}` x `{:?}`",
         display.size().lines(),
         display.size().cols()
     );
```

Comment:
The `info!("Goodbye.");` still has punctuation at the end. `Goodbye!` would be fine, but I don't think the full stop is in line with the style enforced in this PR?

Reply:
Ha, I was kinda hoping to let this one sneak through. It feels right, and I don't think I want it to be an exclamation.

I'm happy to remove the period in favor of consistent consistency ;)

Reply:
I think that would be the right choice.